test/same-day-ach-ppd-write/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/moov-io/ach"
+	"github.com/moov-io/base"
+)
+
+func main() {
+	// Example transfer to write an ACH PPD file to send/credit a external institutions account
+	// Important: All financial institutions are different and will require registration and exact field values.
+
+	// Set originator bank ODFI and destination Operator for the financial institution
+	// this is the funding/receiving source of the transfer
+	fh := ach.NewFileHeader()
+	fh.ImmediateDestination = "231380104"          // Routing Number of the ACH Operator or receiving point to which the file is being sent
+	fh.ImmediateOrigin = "121042882"               // Routing Number of the ACH Operator or sending point that is sending the file
+	fh.FileCreationDate = base.NewTime(time.Now()) // Today's Date
+	fh.ImmediateDestinationName = "Federal Reserve Bank"
+	fh.ImmediateOriginName = "My Bank Name"
+
+	// BatchHeader identifies the originating entity and the type of transactions contained in the batch
+	bh := ach.NewBatchHeader()
+	bh.ServiceClassCode = ach.CreditsOnly
+	bh.CompanyName = "Name on Account" // The name of the company/person that has relationship with receiver
+	bh.CompanyIdentification = fh.ImmediateOrigin
+	bh.StandardEntryClassCode = ach.PPD
+	bh.CompanyEntryDescription = "REG.SALARY" // will be on receiving accounts statement
+	// Same Day ACH
+	bh.EffectiveEntryDate = base.NewTime(time.Now())
+	bh.CompanyDescriptiveDate = "SD1300"
```

Comment:
Can you add this comment above this field. 
```
ODFIs at their discretion may require their Originators to further show intent for same-day settlement
using an optional, yet standardized, same-day indicator in the Company Descriptive Date field. The
Company Descriptive Date field (5 record, field 8) is an optional field with 6 positions available (positions
64-69). Currently, the Rules provide that the “Originator establishes this field as the date it would like to
see displayed to the Receiver for descriptive purposes.” NACHA recommends that, as desired, the content of this field be formatted using the convention “SDHHMM”, where the “SD” in positions 64- 65 denotes the intent for same-day settlement, and the hours and minutes in positions 66-69 denote the desired settlement time using a 24-hour clock. When electing to use this convention, the ODFI would validate that the field contains either.
```

Reply:
yes